internal/app/feed/export_feeds.go

```diff
@@ -116,3 +116,20 @@ func ExportFeeds(v *viper.Viper, apiClient api.APIClient, exporter Exporter) err
 
 	return nil
 }
+
+func ExportInspectionReports(v *viper.Viper, apiClient api.APIClient, exporter *ReportExporter) error {
+	logger := util.GetLogger()
+	ctx := context.Background()
+
+	feed := GetFeeds(v)[0]
+	err := feed.Export(ctx, apiClient, exporter)
```

Comment:
Do we need the full inspections export to do this? Or can we just utilise the same APIs as what @phanikumarpatchigolla is using in https://github.com/SafetyCulture/iauditor-exporter/pull/32.

The file name should already be supplied by a header on the download request.

Reply:
After looking at the code I don't think it would be a very straightforward task.
In `internal/app/inspections/inspections.go` the logic for getting list of inspection and downloading inspection json is tied together.
Overall the amount of time taken to load inspection feed compared to the time taken to download the reports in minimal .